CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,99 @@
+  not be communicating in someone else's primary language.
+
+* **Be respectful**: Not all of us will agree all the time, but disagreement is
+  no excuse for poor behavior and poor manners. We might all experience some
+  frustration now and then, but we cannot allow that frustration to turn into a
+  personal attack. It’s important to remember that a community where people
+  feel uncomfortable or threatened is not a productive one.
+
+* **Be careful in the words that we choose**: We are a community of
+  professionals, and we conduct ourselves professionally. Be kind to others. Do
+  not insult or put down other participants. Harassment and other exclusionary
+  behavior aren't acceptable. This includes, but is not limited to: Violent
+  threats or language directed against another person, Discriminatory jokes and
+  language, Posting sexually explicit or violent material, Posting (or
+  threatening to post) other people’s personally identifying information
+  (“doxing”), Personal insults, especially those using racist or sexist terms,
+  Unwelcome sexual attention, Advocating for, or encouraging, any of the above
+  behavior, Repeated harassment of others. In general, if someone asks you to
+  stop, then stop.
+
+* **Try to understand why we disagree**: Disagreements, both social and
+  technical, happen all the time. It is important that we resolve disagreements
+  and differing views constructively. Remember that we’re different. Diversity
+  contributes to the strength of our community, which is composed of people
+  from a wide range of backgrounds. Different people have different
+  perspectives on issues. Being unable to understand why someone holds a
+  viewpoint doesn’t mean that they’re wrong. Don’t forget that it is human to
+  err and blaming each other doesn’t get us anywhere. Instead, focus on helping
+  to resolve issues and learning from mistakes.
+
+### Diversity Statement
+
+We encourage everyone to participate and are committed to building a community
+for all. Although we will fail at times, we seek to treat everyone both as
+fairly and equally as possible. Whenever a participant has made a mistake, we
+expect them to take responsibility for it. If someone has been harmed or
+offended, it is our responsibility to listen carefully and respectfully, and do
+our best to right the wrong.
+
+Although this list cannot be exhaustive, we explicitly honor diversity in age,
+gender, gender identity or expression, culture, ethnicity, language, national
+origin, political beliefs, profession, race, religion, sexual orientation,
+socioeconomic status, and technical ability. We will not tolerate
+discrimination based on any of the protected characteristics above, including
+participants with disabilities.
+
+### Reporting Issues
+
+If you experience or witness unacceptable behavior—or have any other concerns,
+please report it by contacting us by contacting Dave Clements
```

Comment:
Enthusiastic +1

tir. 18. aug. 2015, 15.09 skrev Nate Coraor <notifications@github.com>:

> In CODE_OF_CONDUCT.md
> <https://github.com/galaxyproject/galaxy/pull/608#discussion_r37301929>:
>
> > +fairly and equally as possible. Whenever a participant has made a mistake, we
> > +expect them to take responsibility for it. If someone has been harmed or
> > +offended, it is our responsibility to listen carefully and respectfully, and do
> > +our best to right the wrong.
> > +
> > +Although this list cannot be exhaustive, we explicitly honor diversity in age,
> > +gender, gender identity or expression, culture, ethnicity, language, national
> > +origin, political beliefs, profession, race, religion, sexual orientation,
> > +socioeconomic status, and technical ability. We will not tolerate
> > +discrimination based on any of the protected characteristics above, including
> > +participants with disabilities.
> > +
> > +### Reporting Issues
> > +
> > +If you experience or witness unacceptable behavior—or have any other concerns,
> > +please report it by contacting us by contacting Dave Clements
>
> Unfortunately I've missed the opportunity to fix prior to votes, so I'll
> just have to reopen - but I'll leave this one up a bit in case any other
> changes pop up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/galaxyproject/galaxy/pull/608/files#r37301929>.
>